kernel/userspace.c

```diff
@@ -342,6 +350,7 @@ static void unref_check(struct _k_object *ko)
 		k_stack_cleanup((struct k_stack *)ko->name);
 		break;
 	default:
+		/* Nothing to do */
```

Comment:
Uh, yeah, that's what an empty case means. :)  But sure.  If you have to put the break there you might as well document what it's (not) doing.